src/main/resources/config.yml

```diff
@@ -109,3 +109,17 @@ sidebar:
   bottom: ""
   # If the sidebar message should overwrite lines in the sidebar when it is full
   overwrite: false
+
+moderation:
+  # If the moderation commands/enforcement should be used
+  enabled: true
+  
+  # Link to a rules website to display on kick screen
+  rules-link: ''
+  
+  # If the rules link is visible on kick screen
+  rules-link-visible: true
```

Comment:
No need for this, just check if `rules-link` is defined.

Reply:
Ah good point. Will adjust